quartz/components/Search.tsx

```diff
@@ -3,8 +3,18 @@ import style from "./styles/search.scss"
 // @ts-ignore
 import script from "./scripts/search.inline"
 
-export default (() => {
+export interface SearchOptions {
+  enablePreview: boolean
+}
+
+const defaultOptions: SearchOptions = {
+  enablePreview: false,
```

Comment:
```suggestion
  enablePreview: true,
```

this is awesome, let's default it!!